niworkflows/interfaces/mni.py

```diff
@@ -161,42 +197,129 @@ def _get_ants_args(self):
                 'write_composite_transform': True,
                 'initial_moving_transform': self.inputs.initial_moving_transform}
 
+        """
+        Moving image handling
```

Comment:
I'd love to see here one of those @effigies' truth tables. I feel we can make all these branches simpler.

| `inputs.moving_mask` | `inputs.explicit_masking` | `inputs.lesion_mask` | action                                                                                                                  |
|----------------------|---------------------------|----------------------|-------------------------------------------------------------------------------------------------------------------------|
| True                 | True                      | True                 | Update `moving_image` after applying `apply_mask` Set `moving_image_mask` applying `create_cfm` with `global_mask=True` |
| True                 | True                      | False                | Update `moving_image` after applying `apply_mask`                                                                       |
| True                 | False                     | True                 | Set `moving_image_mask` applying `create_cfm` with `global_mask=False`                                                  |
| True                 | False                     | False                | `args['moving_image_mask'] =self.inputs.moving_mask`                                                                    |
| False                | *                         | True                 | Set `moving_image_mask` applying `create_cfm` with `global_mask=True`                                                   |
| False                | *                         | False                | Nothing                                                                                                                 |

Reply:
We can just add the table into the comment for now and try to simplify in a future.